src/tools/tidy/src/deps.rs

```diff
@@ -6,47 +6,53 @@ use std::path::Path;
 
 /// These are licenses that are allowed for all crates, including the runtime,
 /// rustc, tools, etc.
+#[rustfmt::skip]
```

Comment:
This one and lower need to keep comments in vertical line, otherwise rustfmt will break it.